cmd/kube-apiserver/app/server.go

```diff
@@ -168,32 +128,44 @@ func CreateServerChain(completedOptions completedServerRunOptions, stopCh <-chan
 		return nil, err
 	}
 
-	kubeAPIServerConfig, insecureServingInfo, serviceResolver, pluginInitializer, admissionPostStartHook, err := CreateKubeAPIServerConfig(completedOptions, nodeTunneler, proxyTransport)
-	if err != nil {
-		return nil, err
+	// create generic apiserver config, shallow copied to all delegated apiservers below
+	genericConfig, versionedInformers, insecureServingInfo, serviceResolver, pluginInitializer, admissionPostStartHook, storageFactory, securityDefinitions, err := CreateGenericConfig(completedOptions, proxyTransport)
+	recommendedConfig := genericapiserver.RecommendedConfig{
```

Comment:
i thought the `genericapiserver.RecommendedConfig` is only meant for aggregated apiservers

Reply:
It is meant for sane API servers, not only aggregated ones :)